src/main/flight/imu.c

```diff
@@ -723,3 +723,18 @@ bool isUpright(void)
     return true;
 #endif
 }
+
+
+#if defined(USE_ALTHOLD_MODE)
+void imuTransformVectorBodyToEarth(t_fp_vector * v)
+{
+    /* From body frame to earth frame */
+    const float x = rMat[0][0] * v->V.X + rMat[0][1] * v->V.Y + rMat[0][2] * v->V.Z;
+    const float y = rMat[1][0] * v->V.X + rMat[1][1] * v->V.Y + rMat[1][2] * v->V.Z;
+    const float z = rMat[2][0] * v->V.X + rMat[2][1] * v->V.Y + rMat[2][2] * v->V.Z;
+
+    v->V.X = x;
+    v->V.Y = -y;
```

Comment:
(someone can eventually fix coordinate systems used in BF. It will probably result in NED system, but it will remove lots of hacks)

Reply:
Out of current scope.